<!DOCTYPE html><html><head><meta charset="utf-8"><meta name= viewport content="width=device-width, initial-scale=1"><meta http-equiv="X-UA-Compatible" content="IE=edge"><title>Research</title><meta name="description" content="The personal website of Mark (Max) Leiserson, PhD candidate in computer science and computational biology at Brown University."><link rel="stylesheet" href="/assets/css/bootstrap/css/bootstrap.min.css"> <style type="text/css">body{font-size:16px;font-family:'Helvetica Neue',Helvetica,Arial,sans-serif;color:#111;font-weight:300}nav ul{list-style-type:none;padding-left:0;margin-left:0}nav ul li{display:inline;margin-right:15px}div.navbar-collapse{padding-left:0;padding-right:0}
ul.nav{padding-left:0}ul.nav li a{line-height:1;padding-top:0;padding-bottom:0;padding-left:0}ul.nav a:hover,ul.nav a:focus{text-decoration:underline;background:#fff}@media only screen and (max-width:766px){.collapsing,.in{background-color:#4e4779;padding-left:15px;margin-top:0}
.collapsing ul,.in ul{padding-left:15px}.collapsing ul li a,.in ul li a{color:#fff!important;padding:5px}.collapsing ul li a:hover,.in ul li a:hover{color:#4e4779!important}}nav h2{padding:0;margin:0;margin-bottom:5px;color:#4e4779}nav a:hover,nav a:focus{text-decoration:none}
div#latest-news{border:1px solid #eee;padding-left:5px;padding-top:3px;padding-bottom:0;margin-bottom:0;font-size:80%;background:#f5f5f5}div#latest-news p.news-item{margin-left:5px}div#latest-news b{margin-right:5px}div#latest-news .show-news{text-decoration:none;cursor:pointer}
div#latest-news .news-item{display:inline}@media only screen and (max-width:992px){div#latest-news .news-item{display:block}}div#about h5{margin-top:0;padding-top:0}div#about-details{padding-top:39px;font-size:80%}@media only screen and (max-width:992px){div#about-details{padding-top:0}
}div#about-details img{max-width:160px}div#recent-events ul{list-style-type:none;padding-left:0;margin-left:0}div#software ul{list-style-type:none;padding-left:0;margin-left:0}.list-group-item:hover{background:#f2f2f2;cursor:pointer}p.img-caption{font-size:90%}
@media only screen and (max-width:992px){p.img-caption{padding-bottom:20px}div#research img{margin-top:20px}.col-img{width:100%}}.vcenter{display:inline-block;vertical-align:middle;float:none}.label-as-badge{border-radius:1em!important;display:inline-block!important}
.label-as-badge a{color:#fff}.popover{background:#fff}.text-xs-left{text-align:left}.text-xs-right{text-align:right}.text-xs-center{text-align:center}.text-xs-justify{text-align:justify}@media(min-width:768px){.text-sm-left{text-align:left}.text-sm-right{text-align:right}
.text-sm-center{text-align:center}.text-sm-justify{text-align:justify}}@media(min-width:992px){.text-md-left{text-align:left}.text-md-right{text-align:right}.text-md-center{text-align:center}.text-md-justify{text-align:justify}}@media(min-width:1200px){.text-lg-left{text-align:left}
.text-lg-right{text-align:right}.text-lg-center{text-align:center}.text-lg-justify{text-align:justify}}.icon{display:inline-block;width:16px;height:16px;vertical-align:middle}.icon svg path{fill:#828282}.contact-list,.social-media-list{list-style:none;margin-left:0;padding-left:0}</style><link rel="canonical" href="http://mdml.github.io/research/"></head><body><header><div class="container" style="margin-top:35px"><div class="navbar"> <nav><div class="navbar-header"> <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-to-collapse" style="background:#4e4779;margin-top:0"> <span class="sr-only">Toggle navigation</span> <span style="background:#fff" class="icon-bar"></span> <span style="background:#fff" class="icon-bar"></span> <span style="background:#fff" class="icon-bar"></span> </button> <a href="/"><h2> Max Leiserson<small><sup> <a href="#" data-toggle="popover" data-trigger='focus' data-placement="right" data-content="My name is often a source of confusion. My real name is Mark D.M. Leiserson, but everyone calls me Max." onclick="javascript:return false"> * </a></sup></small></h2> </a></div> <br class="hidden-xs"/><br class="hidden-xs"/><div class="collapse navbar-collapse" id="navbar-to-collapse"><ul class="nav navbar-nav"><li><a href="/">HOME</a></li><li><a href="/assets/docs/cv.pdf" target="_new">CV</a></li><li><a href="/research">RESEARCH</a></li><li><a href="/publications">PUBLICATIONS</a></li><li><a href="/news">NEWS</a></li></ul></div> </nav></div><hr style="clear:both"/></div> </header><div class="container" id="content"><div id="research"><div class="row"><div class="col-lg-5 col-md-6 vcenter"><h3 style="margin-top:0px;padding-top:0px;">Research</h3><h4>Research statement</h4><p class="caption"> The last decade has seen an unprecedented rise in biolgical data. While the Human Genome Project cost over one billion dollars to complete, less than ten years later genomes are routinely sequenced for < $10,000. Thus data analysis, instead of data production, has become an urgent challenge.</p><p class="caption"> My research interests are in developing algorithms and mathematical models for understanding biological processes, in particular those related to disease. There are many challenges in analyzing modern biological data sets e.g. scale (every genome is >1Gb of information), the relatively low signal-to-noise ratio, and the complexity of biology. As a result, it is imperative that new approaches utilize both biological and quantitative methods and expertise. My goal is to be at the heart of this process, and to develop new methods and algorithms that are efficient, scalable, and easy-to-use (by biologists and computer scientists alike), but that capture the complexity of biological processes.</p></div><div class="col-lg-5 col-md-6 text-center vcenter col-img"> <a title="By Ben Moore (Own work) [CC BY-SA 3.0 (http://creativecommons.org/licenses/by-sa/3.0)], via Wikimedia Commons" href="http://commons.wikimedia.org/wiki/File%3AHistoric_cost_of_sequencing_a_human_genome.svg"> <img alt="Historic cost of sequencing a human genome" src="/assets/img/research/cost-of-dna-sequencing.svg" style="max-height:350px" class="img-responsive img-thumbnail"/> </a><p class="img-caption"> The cost of DNA sequencing over the last 15 years (image from Wikimedia Commons, data from NHGRI).</p></div></div><hr style="clear:both"/><div class="row"><div class="col-lg-5 col-md-6 vcenter"><h4>Cancer genomics</h4><p class="caption"> Advances in next-generation sequencing technology now allow for the sequencing of hundreds of tumors (cancer genomes) of various cancer types. Projects such as <a href="http://cancergenome.nih.gov" target="_new">The Cancer Genome Atlas</a> and the <a href="http://www.icgc.org/" target="_new">International Cancer Genome Consortium</a> have already sequenced thousands of tumor-normal pairs, where both a tumor and adjacent healthy tissue are sequenced in the same individual. These tumor-normal matched pairs allow for a whole-exome or whole-genome comparison -- comprising billions of bases of DNA -- in hundreds of tumors to identify cancer mutations. The goal of such a comparison is to discover the underlying genetic cause(s) of cancer. However, a major challenge in analyzing cancer genomes is that they contain hundreds to thousands of mutations that play no role in cancer, but simply arise randomly through cancer cell development. Due to the size and complexity of these datasets -- billions of base pairs in hundreds or thousands of tumors -- computational methods for analyzing cancer genomes are an urgent priority.</p><p class="caption"> My research in the area of cancer genomics has focused on identifying <i>driver mutations</i>, or those that are required for cancer. In particular, I am interested in identifying <i>combinations</i> of driver mutations from a cohort of tumor samples. Cancer is a heterogeneous disease, as different tumors of the same cancer type often harbor different driver mutations. One cause for this heterogeneity is that mutations in cancer target genetic pathways, or sets of genes that interact to perform a biological function, instead of targeting individual genes. Thus the problem of identifying the mutations that cause cancer can be formulated as identifying sets of genes that are recurrently mutated across cancer.</p></div><div class="col-lg-5 col-md-5 text-center vcenter"> <img class="img-responsive img-thumbnail" src="/assets/img/research/clonal-theory.png"/><p class="img-caption"> The clonal theory of cancer (Nowell 1976) posits that cancer is partially a result of mutations that accumulate over time. A key subset of these mutations are <i>driver</i> mutations, which are responsible for causing cancer.</p></div></div><div class="row"><div class="col-lg-5 col-md-6 vcenter"><p class="caption"> My research has focused on identifying combinations of driver mutations (e.g. sets of mutated genes). To date, I have led development of three major algorithms with my advisor Dr. Benjamin J. Raphael:<ul><li> CoMEt [1]: identifies <u>c</u>ombinations <u>o</u>f <u>m</u>utually <u>e</u>xclusive al<u>t</u>erations in cancer using a statistical exact test to score combinations, and a Markov chain Monte Carlo (MCMC) algorithm to sample from the distribution of solutions.</li><li> HotNet2 [2]: identifies significantly mutated subnetworks of a protein-protein interaction network using a heat diffusion process that simultaneously analyzes a protein's mutations and interactions.</li><li> Multi-Dendrix [3]: identifies multiple groups of exclusive mutations using an integer linear program.</li></ul> Dr. Raphael and I have also led the development of the MAGI platform for visualizing and annotation combinations of mutations [4].</p></div><div class="col-lg-5 col-md-6 text-center vcenter"> <img class="img-responsive img-thumbnail" src="/assets/img/research/hotnet2-nutshell.png"/><p class="img-caption"> The HotNet2 algorithm in a nutshell.</p></div></div><hr style="clear:both"/><div class="row"><div class="col-lg-5 col-md-6 vcenter"><h4>Genetic fault-tolerance in yeast</h4><p class="caption"> Due to advances in robotics and biochemistry, the last ten to fifteen years have witnessed the development of high-throughput experiments that can quickly measure hundreds of interactions between genes and/or their protein products. One such interaction, called a physical interaction, indicates whether the protein products of a pair of genes bind to another physically in vivo. Another such interaction, <i>epistasis</i>, measures the effect one gene has on the expression of another. These genetic interactions can be used to predict functional features of different genes (which, if correct, can later be validated through molecular experiments). A synthetic lethal interaction between genes A and B is a type of genetic interaction results if mutations to either A or B do not significantly affect the viability of an organism, while simultaneous mutations to both A and B render the organism inviable. Synthetic lethal interactions between genes have been interpreted as an indication of related, or redundant, function. The intuition behind this interpretation is that an organism may have multiple genes that perform the same, vital function, such that only removing more than one gene renders the organism inviable.</p></div><div class="col-lg-5 col-md-6 text-center vcenter col-img"> <a title="By AthenaPendergrass (Own work) [CC BY-SA 4.0 (http://creativecommons.org/licenses/by-sa/4.0)], via Wikimedia Commons" href="http://commons.wikimedia.org/wiki/File%3ASynthetic_lethality.jpg"> <img alt="Synthetic lethality" src="//upload.wikimedia.org/wikipedia/commons/thumb/d/d5/Synthetic_lethality.jpg/512px-Synthetic_lethality.jpg" class="img-responsive img-thumbnail" style="max-height:350px"/> </a><p class="img-caption"> A schematic of synthetic lethality (from Wikimedia Commons).</p></div></div><div class="row"><div class="col-lg-5 col-md-6 vcenter"><p class="caption"> My work has focused on the between-pathway model (BPM), which is comprised of two putatively redundant genetic pathways. The original formulation of the BPM by Kelley and Ideker in 2005 was as a network motif in physical and genetic protein-protein interaction graphs, where vertices are genes/proteins and edges connect genes/proteins that share a physical/genetic interaction. The BPM consists of two gene sets U and V such that there are<ul><li> many physical interactions within U and V (and few between), indicating that U and V are separate genetic pathways; and,</li><li> many genetic interactions between U and V (and few within), indicating that U and V perform redundant functions or are in a compensatory relationship.</li></ul> I worked on two projects related to the BPM in baker's yeast, in collaboration with <a href="http://www.cs.tufts.edu/~hescott" target="_new">Ben Hescott</a>, <a href="http://www.cs.tufts.edu/~cowen" target="_new">Lenore Cowen</a>, <a href="http://www.cs.tufts.edu/~slonim" target="_new">Donna Slonim</a>, and Diana Tatar at Tufts University. In the first, we used knockout expression data to evaluate BPMs [5]. For the second project, we used pairwise measures of epistasis in sets of genes to identify new BPMs. [6]</p></div><div class="col-lg-5 col-md-6 text-center vcenter"> <img alt="Schematic of the between-pathway model" src="/assets/img/research/example-bpm.svg" style="max-height:350px"/><p class="img-caption"> A schematic of the between-pathway model, with two partitions U and V each with many physical interactions within and genetic (e.g. synthetic lethal) interactions between.</p></div></div><div class="row"><div class="col-lg-8 col-md-8"><h4>References</h4><ol><li> M.D.M. Leiserson*, H-T. Wu*, F. Vandin, B.J. Raphael. (2015) CoMEt: A Statistical Approach to Identify Combinations of Mutually Exclusive Alterations in Cancer. <i>RECOMB 2015</i> (to appear).</li><li> M.D.M. Leiserson*, F. Vandin*, H-T. Wu, J.R Dobson, J.V. Eldridge, J.L. Thomas, A. Papoutsaki, Y. Kim, B. Niu, M. McLellan, M.S. Lawrence, A. Gonzalez-Perez, D. Tamborero, Y. Cheng, G.A. Ryslik, N. Lopez-Bigas, G. Getz, L. Ding, B.J. Raphael. (2015) Pan-cancer network analysis identifies combinations of rare somatic mutations across pathways and protein complexes. <i>Nature Genetics</i>, 47(<b>2</b>):106- 114.</li><li> M.D.M. Leiserson, D. Blokh, R. Sharan*, B. Raphael*. (2013) Simultaneous Identification of Multiple Driver Pathways in Cancer. <i>PLoS Comp Biol</i>, 9(<b>5</b>):e1003054.</li><li> M.D.M. Leiserson, C.C. Gramazio, J. Hu, H-T. Wu, D.H. Laidlaw, B.J. Raphael (2015) MAGI: A Platform for Interactive Visualization and Collaborative Annotation of Combinations of Genetic Aberrations. (Unpublished.)</li><li> B. Hescott, M.D.M. Leiserson, D. Slonim, L. Cowen. (2009) Evaluating Between-Pathway Models with Expression Data. Journal of Computational Biology 17(3):443-457.</li><li> M.D.M. Leiserson, D. Tatar, L. Cowen, B. Hescott. (2011) Inferring Mechanisms of Compensation from E-MAP and SGA Data Using Local Search Algorithms for Max Cut. <i>Journal of Computational Biology</i>, 18(<b>11</b>):1399-1409. Also <i>RECOMB 2011</i>.</li></ol> * equal contribution</div></div></div></div><footer><div class="container" style="font-size:90%"><hr/><div class="col-md-4 text-left"><p class="text"> Created with <a href="http://jekyllrb.com" target="_new">Jekyll</a> and hosted on <a href="http://github.com" target="_new">GitHub</a>.</p></div><div class="col-md-5 text-md-center"><p class="text">© 2015 Max Leiserson. Some rights reserved.</p></div><div class="col-md-3 text-md-right"><p class="text">Updated: March 31, 2015</p></div></div> </footer></body></html> <script src="/assets/js/jquery.min.js"></script><script src="/assets/js/bootstrap.min.js"></script><script>!function(e,t,a,n,o,c,g){e.GoogleAnalyticsObject=o,e[o]=e[o]||function(){(e[o].q=e[o].q||[]).push(arguments)},e[o].l=1*new Date,c=t.createElement(a),g=t.getElementsByTagName(a)[0],c.async=1,c.src=n,g.parentNode.insertBefore(c,g)}(window,document,"script","//www.google-analytics.com/analytics.js","ga"),ga("create","UA-17382912-1","auto"),ga("send","pageview"),$(function(){$('[data-toggle="popover"]').popover({html:!0,container:"body"})})</script>